tests/testthat/test-read_redcap.R

```diff
@@ -19,7 +19,8 @@ test_that("read_redcap works for a classic database with a nonrepeating instrume
       # suppress expected warning
       suppressWarnings(classes = c(
         "field_missing_categories",
-        "empty_parse_warning"
+        "empty_parse_warning",
+        "duplicate_labels"
```

Comment:
Is there a better way we could handle the growing number of warnings we have to account for in our tests? We can't just reference `classes = "REDCapTidieR_cond"`?

Reply:
Ha I was thinking this too. We can do that as long as we're fine silencing *other* `REDCapTidieR` warnings besides the ones we expect.

An alternative is just to move our warning-generating cases to a new REDCap so our main test REDCap will be warning-free. Not sure if that's better or worse

Reply:
Hm, it might pay to look into cleaning up our test databases like how Will has 50 for unique test cases. But that will also be a semi-large lift.

Not anything necessary for this PR but something for us to throw on the pile of considerations ✨